foxy/distribution.yaml

```diff
@@ -395,6 +395,21 @@ repositories:
       url: https://github.com/ros/bond_core.git
       version: dashing-devel
     status: maintained
+  carla_msgs:
+    doc:
+      type: git
+      url: https://github.com/carla-simulator/ros-carla-msgs.git
+      version: release
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/carla-simulator/ros-carla-msgs-release.git
+      version: 1.3.0-1
+    source:
+      type: git
+      url: https://github.com/carla-simulator/ros-carla-msgs.git
+      version: release
```

Comment:
It doesn't look the `release` branch is the correct branch to track. The CMakeLists.txt looks like it doesn't have the necessary changes for ROS 2. I guess either the `release` branch should be updated or this should be changed to `master`.

Reply:
Updated to track `master` branch